### PR TITLE
Add support for Rails 5.2

### PR DESCRIPTION
**Pull request by jszymanowski**
/domain @samandmoore @jmileham @smudge @Jaco-Pretorius
/platform @samandmoore @jmileham @smudge @Jaco-Pretorius

### CODE REVIEW

File: Appraisals [hunk 1]
```diff
@@ -9,3 +9,7 @@ end
 appraise 'rails-5-1' do
   gem 'rails', '~> 5.1.0'
 end
+
+appraise 'rails-5-2' do
+  gem 'rails', '~> 5.2.0'
+end
```
Comment (smudge) [inline Appraisals]: Minor - newline at EOF

**Comment by smudge** [comment]
<< domain LGTM && platform TAFN -- Looks like 4.2-5.0 are having issues with the latest sqlite gem, might need to pin it to a lower version in their respective appraisals. Also, probably fine to remove the Rails 4.2 appraisal now. Probably 5.0 too unless it's a quick fix.

**Comment by samandmoore** [comment]
Yea. I suggest we rip out any versions that we don't use.

**Comment by jszymanowski** [comment]
Looks like the sqlite gem issues are resolved now after pinning the versions.  Nanda is on Rails 4.2 so I left it in.

bump

**Comment by smudge** [comment]
platform LGTM 👍

**Pull request merged**